pkg/file/error.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package file
+
+// ErrAborted should be returned whenever a file operation is terminated 
+// before it has completed.
+type ErrAborted struct {
```

Comment:
Error types should be named with Error suffix, in this case AbortError. Err prefix is related to error variables.